### ISSUE TITLE
Compiled version of code not working (Babel compilation working)

### ISSUE THREAD

**Issue by usmanmani1122** [swc-project/swc#6040]
### Describe the bug

The following compiled code is not working:
```
window.getSelection?.().empty?.()
```

|   Compilation (SWC)    |    Working    |
| :---------: | :---------: |
| <img width="1728" alt="image" src="https://user-images.githubusercontent.com/57341641/193930308-8859d18f-a932-49ca-aaaa-1bded6bb9288.png"> | <img width="552" alt="image" src="https://user-images.githubusercontent.com/57341641/193930403-71f92f3a-ae40-4c1c-b836-d378327699d6.png"> |

The same code used to work when using babel compiler:

|   Compilation (Babel)    |    Working    |
| :---------: | :---------: |
| <img width="1728" alt="image" src="https://user-images.githubusercontent.com/57341641/193930730-fd821b88-3f0b-44fd-9e20-88bd28826e81.png"> | <img width="552" alt="image" src="https://user-images.githubusercontent.com/57341641/193930826-88dea119-d4b3-4e3d-b1c1-4055ac14d583.png"> |

### Input code

```typescript
window.getSelection?.().empty?.()
```

### Config

```json
Not using any
```

### Expected behavior

The compiled code should work

### Actual behavior

The compiled code throws `TypeError: Illegal invocation` error

### Version

1.3.4

**Comment by iheyunfei** [comment]
It seems fixed in 1.3.24. [playground link](https://play.swc.rs/?version=1.3.24&code=H4sIAAAAAAAAAyvPzEvJL9dLTy0JTs1JTS7JzM%2Bz19PQ1EvNLSipBLG4ANGqKqwiAAAA&config=H4sIAAAAAAAAA0WMSwrDMAxE76K1F02gXeQOPYRwleDiH5ICNcZ3r11cshtm3rwKb7GwVcjIQjySlKj4gQ3IBhTLLiuYjvVqRy%2FUDCjyQToQWW%2FLvc8%2BJaEJGAguur0MmU0hM4lcE8bD%2F8nWXSG9zlFU0JLp53xAuxzz5%2BQ5QeWT2he35aBbuAAAAA%3D%3D).

**Comment by kdy1** [comment]
Thank you for verifying!
